scenes/Restaurant/CustomerBody.gd

```diff
@@ -0,0 +1,88 @@
+		"woozy": [
+			[Vector2(2.48196, -98.051903), Vector2(24.979601, 1.01005), Vector2(-47.211102, -1.90899)],
+			[Vector2(-73.3638, -25.1984), Vector2(-0.830624, -38.093201), Vector2(0.634766, 29.110901)],
+			[Vector2(0.225128, 32.501999), Vector2(-35.993198, -0.577805), Vector2(42.113602, 0.676056)],
+			[Vector2(77.995201, -21.8535), Vector2(-0.801091, 32.944302), Vector2(1.04004, -42.770901)],
+			[Vector2(3.64993, -98.051903), Vector2(47.535599, -1.59304), Vector2(-24.986, 0.837346)]
+		]},
+	{"fatness": 2.0,
+		"woozy": [
+			[Vector2(-0.795074, -123.774002), Vector2(24.9736, 1.14793), Vector2(-54.332802, -2.49744)],
+			[Vector2(-103.721001, -28.409201), Vector2(0.105613, -58.087898), Vector2(-0.072546, 39.901001)],
+			[Vector2(-3.13336, 45.235298), Vector2(-63.802299, 0.391274), Vector2(61.3144, -0.376016)],
+			[Vector2(99.250504, -28.409201), Vector2(-0.06645, 39.297901), Vector2(0.097053, -57.396599)],
+			[Vector2(-0.352325, -123.877998), Vector2(52.4123, -1.59304), Vector2(-24.988501, 0.759509)]
+		]},
+	{"fatness": 3.0,
+		"woozy": [
+			[Vector2(-8.91397, -205.408997), Vector2(24.998699, 0.254314), Vector2(-81.092499, -0.824962)],
+			[Vector2(-153.753006, -56.668701), Vector2(-0.645028, -82.339104), Vector2(0.43048, 54.951599)],
+			[Vector2(-7.45096, 49.156601), Vector2(-83.869598, 1.41249), Vector2(82.6315, -1.39163)],
+			[Vector2(138.363998, -50.816601), Vector2(-0.945125, 49.6637), Vector2(1.22209, -64.217499)],
+			[Vector2(-8.91397, -204.921997), Vector2(88.370598, -1.59304), Vector2(-24.996, 0.450598)]
+		]},
+	{"fatness": 5.0,
+		"woozy": [
+			[Vector2(-15.1975, -342.556), Vector2(24.9942, -0.537486), Vector2(-120.339996, 2.58783)],
+			[Vector2(-262.069, -87.764297), Vector2(-2.46623, -173.910004), Vector2(1.31159, 92.488998)],
+			[Vector2(-29.1206, 90.612198), Vector2(-137.335999, -0.624495), Vector2(163.981003, 0.745659)],
+			[Vector2(236.408005, -78.804703), Vector2(-0.799582, 82.650398), Vector2(1.55683, -160.925003)],
+			[Vector2(-14.9055, -342.556), Vector2(128.186996, -2.16184), Vector2(-24.9965, 0.421561)]
+		]},
+	{"fatness": 10.0,
+		"woozy": [
+			[Vector2(-1.29367, -658.921021), Vector2(24.996201, 0.436254), Vector2(-186.572998, -3.25622)],
+			[Vector2(-494.141998, -156.332993), Vector2(-4.48658, -290.76001), Vector2(3.44123, 223.014008)],
+			[Vector2(-18.8258, 204.050003), Vector2(-219.143005, -2.18805), Vector2(241.903, 2.4153)],
+			[Vector2(497.39801, -131.007996), Vector2(2.04043, 215.119003), Vector2(-3.31852, -349.867004)],
+			[Vector2(0.654327, -658.921021), Vector2(194.419998, -4.10986), Vector2(-24.994499, 0.528361)]
+		]}
+	]
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	if editing:
+		for i in curve.get_point_count():
+			print("[Vector2%s, Vector2%s, Vector2%s]," % [curve.get_point_position(i), curve.get_point_in(i), curve.get_point_out(i)])
+
+func set_fatness(fatness):
+	_fatness = fatness
+	
+	if editing:
+		# don't wipe out the curve
```

Comment:
Add a better comment; why don't we want to 'wipe out the curve'? what does this mean?

Reply:
Improved comment